include/net/ieee802154_radio.h

```diff
@@ -54,14 +54,44 @@ struct ieee802154_filter {
 /* @endcond */
 };
 
+/** IEEE802.15.4 Transmission mode. */
+enum ieee802154_tx_mode {
+	/** Transmit packet immediately, no CCA. */
+	IEEE802154_TX_MODE_PLAIN,
```

Comment:
```IEEE802154_TX_MODE_NO_CCA``` to be more explicit?

Reply:
I wanted to make `_NO_CCA` implicit, so I don't need to add it to the delayed variant as well. But  I can see that `_PLAIN` might not sound too good in terms of transmission. How about `IEEE802154_TX_MODE_DIRECT`/`IEEE802154_TX_MODE_IMMEDIATE`/`IEEE802154_TX_MODE_INSTANT`? Any other ideas?